include/wil/registry.h

```diff
@@ -223,7 +223,7 @@ namespace reg
     //
 #if defined(WIL_ENABLE_EXCEPTIONS)
 
-#if defined(_STRING_) || defined(WIL_DOXYGEN)
+#if WIL_USE_STL || defined(WIL_DOXYGEN)

```

Comment:
`WIL_DOXYGEN` defines `WIL_USE_STL` so the second condition will never be relevant.  If it is true then the `WIL_USE_STL` check will already be true, rendering it moot.

Reply:
See the other comment. The purpose of `WIL_DOXYGEN` is to short-circuit the logic so that Doxygen includes the code in its documentation pass. The purpose is to not think too hard about whether or not the surrounding logic will evaluate to true